third_party/unix/README

```diff
@@ -0,0 +1,11 @@
+The code from Package unix is generated by running GOOS=linux GOARCH=amd64 ./mkall.sh from [golang/sys/internal-branch.go1.17-vendor](https://github.com/golang/sys/tree/internal-branch.go1.17-vendor)
```

Comment:
Please name the file README.md so that it will be rendered as markdown by github.

Please wrap the lines like other documents.

Reply:
README.md updated.